### PR TITLE
RHIROS-180 fixes 404 error shown when no recommendations

### PR DESCRIPTION
**Pull request by apuntamb**

### CODE REVIEW

File: ros/api/v0/recommendations.py [hunk 1]
```diff
@@ -63,5 +63,4 @@ def get(self, host_id):
                   'meta': {'count': len(recommendations_list)}
             }
         else:
-            abort(404, message="host with id {} doesn't have any recommendation"
-                  .format(host_id))
+            recommendations_list = []
```
Comment (kgaikwad) [inline ros/api/v0/recommendations.py:66]: @apuntamb - what is the expected return value in this case?

Comment (apuntamb) [reply ros/api/v0/recommendations.py]: return value should be common for both cases. added a commit for the same.

File: ros/api/v0/recommendations.py [hunk 2]
```diff
@@ -56,11 +56,11 @@ def get(self, host_id):
                     for skey in rules_columns:
                         recommendation[skey] = eval("f'{}'".format(rule_dict[skey]))
                     recommendations_list.append(recommendation)
+        else:
+            recommendations_list = []
```
Comment (kgaikwad) [inline ros/api/v0/recommendations.py:60]: No need of `else` block as `recommendations_list` already initialized to empty array.

**Review by kgaikwad — APPROVED** [review]
Thanks @apuntamb! LGTM :+1:

**Pull request merged**